config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# Configure parameters to be filtered from the log file. Use this to limit dissemination of
+# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
+# notations and behaviors.
+Rails.application.config.filter_parameters += [
```

Comment:
what does this do?

Reply:
limits dissemination of sensitive information in logs. For example value of password attribute will be shown as "FILTERED" if it is mentioned in the above config.